sceptre/plan/actions.py

```diff
@@ -212,7 +212,11 @@ def launch(self) -> StackStatus:
 
         if existing_status == "PENDING":
             status = self.create()
-        elif existing_status in ["CREATE_FAILED", "ROLLBACK_COMPLETE"]:
+        elif existing_status in [
```

Comment:
the docs for the [create command](https://docs.sceptre-project.org/latest/docs/cli.html?highlight=CHANGE_SET_NAME#sceptre-create) provides info about change sets while the doc and CLI help for the `launch` command does not.  Could you please add some documentation to launch?

Reply:
@zaro0508  I've had a look and I'm afraid I'm not sure exactly what the issue is. My understanding is the create command can also create a change set; the launch command I assume cannot. The docs look right to me?

Reply:
Am I missing something?  The launch command doesn't create the change set however doesn't this enable the launch command to deploy change sets?

Reply:
@zaro0508 Yes, I think you could be misunderstanding what I am changing here. All that changes is an additional state, `REVIEW_IN_PROGRESS`, is added to the list of possible existing statuses that launch needs to be able to handle. Actually, the fact that it is currently not handled is arguably a bug anyway and this change to the launch method therefore is just a trivial bugfix. 

> The launch command doesn't create the change set however doesn't this enable the launch command to deploy change sets?

No. If launch is used on a stack that has a change set, that stack will exist and be in state `REVIEW_IN_PROGRESS`. The launch command will not execute that change set; it will delete this 'placeholder' stack that gets created by the create changeset command, and then launch a new one in its place.

If the caller doesn't want this, they would instead use the changeset execute command rather than launch.

So all will continue to behave as documented, and as expected.